index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+
+
+    <title>Jean Michel Basquiat </title>
+    
+    <!--
```

Comment:
@elysemclark Good job in not using styles in the head. We always want to have a link to our styles like you did above on line 6. Good best practice.